# Controls torque output for motors
# Request:
#   set_enable:
#     - Array containing an on/off for each motor
#     - Must be the same length as the number of motors
#     - On: true; Off: false
# Response:
#   success:
#     - Returns whether the set was successful or not

bool[] set_enable

---

bool success
